File: app/src/main/java/com/queserasera/lostarkhomework/SplashActivity.java
package com.queserasera.lostarkhomework;

import android.app.Activity;
import android.content.Intent;
import android.content.SharedPreferences;
import android.os.Bundle;
import android.os.Handler;

public class SplashActivity extends Activity {
    Handler handler = new Handler();
    private SharedPreferences appData;

    /** Called when the activity is first created. */
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        // 설정값 불러오기
        appData = getSharedPreferences("appData", MODE_PRIVATE);

        handler.postDelayed(new Runnable(){
            @Override
            public void run(){
                showMain();
            }
        }, 1000);
    }
    public void showMain(){
        Intent intent = new Intent(getBaseContext(), MainActivity.class);
        startActivity(intent);
        overridePendingTransition(0, 0);
        finish();
        overridePendingTransition(0, R.anim.fadeout);
    }
}

File: app/src/main/java/com/queserasera/lostarkhomework/SplashActivity.kt
package com.queserasera.lostarkhomework

import android.content.Intent
import android.content.SharedPreferences
import android.os.Bundle
import android.os.Handler
import androidx.appcompat.app.AppCompatActivity

class SplashActivity : AppCompatActivity() {
    private var handler = Handler(mainLooper)
    private var appData: SharedPreferences? = null

    /** Called when the activity is first created.  */
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        // 설정값 불러오기
        appData = getSharedPreferences("appData", MODE_PRIVATE)
        handler.postDelayed({ showMain() }, 1000)
    }

    private fun showMain() {
        val intent = Intent(baseContext, MainActivity::class.java)
        startActivity(intent)
        overridePendingTransition(0, 0)
        finish()
        overridePendingTransition(0, R.anim.fadeout)
    }
}